mkt/lookup/urls.py

```diff
@@ -40,10 +43,11 @@
 # These views all start with transaction ID.
 transaction_patterns = patterns(
     '',
+    url(r'^$', views.transaction_summary,
+        name='lookup.transaction_summary'),
+    url(r'^summary$', views.transaction_summary),
```

Comment:
I'm not sure why we need to keep the old ones around. For a public resource I'd say we need them and set up redirects. But for lookup tool, if they aren't used, I'd vote to just remove it completely.